src/stores/root.ts

```diff
@@ -1468,6 +1478,14 @@ export class RootStore {
 				],
 			},
 			{
+				poolId: '677',
+				currencies: [
+					{
+						coinMinimalDenom: DenomHelper.ibcDenom(
+							[{ portId: 'transfer', channelId: 'channel-169' }],
+							'cw20:juno1y9rf7ql6ffwkv02hsgd4yruz23pn4w97p75e2slsnkm0mnamhzysvqnxaq'
+						),
+						coinDenom: 'BLOCK',
```

Comment:
the structure of the object here is incomplete. Missing the osmo portion and closing brackets.